scripts/cot/botania/flowers/lexicon.zs

```diff
@@ -0,0 +1,286 @@
+<botania:rune:14>,
+<botania:rune:9>,
+<quark:rune:2>,
+<quark:rune:4>,
+<quark:rune:4>,
+<thaumicaugmentation:material:5>,
+]]);
+mods.botania.Lexicon.addRecipeMapping(<botania:specialflower>.withTag({type: "jikanacea"}), "botania.entry.jikanacea", 0);
+
+mods.botania.Lexicon.addEntry("botania.entry.echinacenko", "botania.category.functionalFlowers", <botania:specialflower>.withTag({type: "echinacenko"}));
+mods.botania.Lexicon.setEntryKnowledgeType("botania.entry.echinacenko", "alfheim");
+mods.botania.Lexicon.addTextPage("botania.page.echinacenko0", "botania.entry.echinacenko", 0);
+mods.botania.Lexicon.addPetalPage("botania.page.echinacenko1", "botania.entry.echinacenko", 1, [<botania:specialflower>.withTag({type: "echinacenko"})], [[
+<quark:rune:2>,
+<quark:rune:2>,
+<quark:rune:5>,
+<botania:rune:10>,
+<botania:rune:12>,
+<botania:manaresource:8>,
+]]);
+mods.botania.Lexicon.addRecipeMapping(<botania:specialflower>.withTag({type: "echinacenko"}), "botania.entry.echinacenko", 0);
+
+mods.botania.Lexicon.addEntry("botania.entry.amuileria_kaerunea", "botania.category.functionalFlowers", <botania:specialflower>.withTag({type: "amuileria_kaerunea"}));
+mods.botania.Lexicon.setEntryKnowledgeType("botania.entry.amuileria_kaerunea", "alfheim");
+mods.botania.Lexicon.addTextPage("botania.page.amuileria_kaerunea0", "botania.entry.amuileria_kaerunea", 0);
+mods.botania.Lexicon.addPetalPage("botania.page.amuileria_kaerunea1", "botania.entry.amuileria_kaerunea", 1, [<botania:specialflower>.withTag({type: "amuileria_kaerunea"})], [[
+<botania:manaresource:9>,
+<botania:rune:8>,
+<minecraft:end_rod>,
+<quark:rune:3>,
+<quark:rune:3>,
+<quark:rune:4>,
+<quark:rune>,
+]]);
+mods.botania.Lexicon.addRecipeMapping(<botania:specialflower>.withTag({type: "amuileria_kaerunea"}), "botania.entry.amuileria_kaerunea", 0);
+
+/////////////////
+//Lexicon fixes//
+/////////////////
+mods.botania.Lexicon.removePage("botania.entry.runeAltar", 7);
+mods.botania.Lexicon.removePage("botania.entry.runeAltar", 7);
+mods.botania.Lexicon.removePage("botania.entry.runeAltar", 7);
+mods.botania.Lexicon.removePage("botania.entry.runeAltar", 7);
+mods.botania.Lexicon.removePage("botania.entry.runeAltar", 7);
+mods.botania.Lexicon.removePage("botania.entry.runeAltar", 7);
+mods.botania.Lexicon.removePage("botania.entry.runeAltar", 7);
+mods.botania.Lexicon.removePage("botania.entry.runeAltar", 7);
+mods.botania.Lexicon.removePage("botania.entry.runeAltar", 7);
+mods.botania.Lexicon.removePage("botania.entry.runeAltar", 7);
+mods.botania.Lexicon.removePage("botania.entry.runeAltar", 7);
+mods.botania.Lexicon.removePage("botania.entry.runeAltar", 7);
+mods.botania.Lexicon.removePage("botania.entry.runeAltar", 7);
+mods.botania.Lexicon.removePage("botania.entry.runeAltar", 7);
+mods.botania.Lexicon.removePage("botania.entry.runeAltar", 7);
+mods.botania.Lexicon.removePage("botania.entry.runeAltar", 7);
+mods.botania.Lexicon.removePage("botania.entry.runeAltar", 3);
```

Comment:
Is it intended to 16 times call same function with same arguments? Or its a typo?

Reply:
It is intended. 

When page 7 is removed, page 8 becomes 7, page 9 is 8 and so on. That took me some time to understand how lexicon works since documentation isn't well written